src/main/webapp/jsp/see-all-tasks.jsp

```diff
@@ -0,0 +1,100 @@
+<%@ page import="com.todolist.app.model.Item" %>
+<%@ page import="java.util.Map" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>See all tasks</title>
+    <link rel="stylesheet" href="<%= request.getContextPath() %>/css/see-all-tasks.css">
+    <link href='https://unpkg.com/css.gg@2.0.0/icons/css/more-o.css' rel='stylesheet'>
+</head>
+<body>
+<div class="seeall">
```

Comment:
I'm not an expert in html, but I suppose there should be used dash(-) or underscore(_) to separate the words, e.g. https://gist.github.com/whizark/92ed70d90d262a44db84